src/browser-realm.ts

```diff
@@ -80,7 +81,10 @@ export default function createSecureEnvironment(distortionMap?: Map<SecureProxyT
             let rawError;
             const { message, constructor } = e;
             try {
-                rawError = construct(env.getRawRef(constructor), [message]);
+                const sr: SecureRecord | undefined = WeakMapGet(env.som, constructor);
```

Comment:
unfolding `getRawRef` since it is now removed.

Reply:
For maintainability/readability, how about having our own transpilation/codemod so that we can still use `env.getRawRef` at the source level?

Reply:
ok, will do.